packages/components/molecules/f-restaurant-card/src/components/subcomponents/_tests/IconText.test.js

```diff
@@ -48,41 +48,47 @@ describe('IconText.vue', () => {
         expect(accessibleText).toBe('foo');
     });
 
-    it('when icon is present and hideInTileView is true and in tile mode, hides Icon', () => {
-        // arrange
-        const propsData = {
-            text: 'foo',
-            isListItem: false,
-            hideIconInTileView: true
-        };
+    describe('Icon Visibility', () => {
+        it('should hide icon when "hideIconInTileView" is true and in Tile view', () => {
```

Comment:
you could even go further with the describes and `describe('tile view')` and `describe('list view')` - then the test case is focused soley on the value of `hideIconInTileView`

Reply:
Awesome, on it 😄